src/manipulation.js

```diff
@@ -17,7 +17,7 @@ define([
 
 var
 	rxhtmlTag = /<(?!area|br|col|embed|hr|img|input|link|meta|param)(([\w:]+)[^>]*)\/>/gi,
-	rtagName = /<([\w:]+)/,
+	rtagName = /<([\w:-]+)/,
```

Comment:
> rsingleTag is the test for element creation via document.createElement

This is true

> and therefore shouldn't include ":".

This isn't

> rtagName is for finding the first open tag in an HTML string

This is true

> and therefore should include ":"

This isn't

`rsingleTag` is needed to identify single tag, single tag name might include ":" sign, which could be processed by `createElement` method.

`rtagName` doesn't need to identify xml tags, only html tags, since only html tags need additional wrapping, so it could leave without that symbol.

Re-reading my previous comment, i probably wasn't clear - should we drop the ":" sign in `rtagName` regexp or add support for those tags in core module? Either way, i would like to be consistent